src/main/PDF/Services/CrudServices/UploadPDFService.java

```diff
@@ -40,7 +41,8 @@ public UploadPDFService(
       String filename,
       String fileContentType,
       InputStream fileStream,
-      EncryptionController encryptionController) {
+      EncryptionController encryptionController,
+      String idCategory) {
```

Comment:
can you not have the id category be in the upload pdf service? shouldn't this be in the upload file service?

Reply:
No, because the frontend code uses the pdf service still. Look at our url endpoints, it still uses the pdfcontroller. I didn't work on the code base so if you wanted to stop using the pdf controller you should have migrated it over already

Reply:
But I put it in the file service stuff as you asked right now